elegant/worm_spline.py

```diff
@@ -37,6 +37,8 @@ def pose_from_mask(mask, smoothing=2):
     sx, sy = slices[0]
     cropped = mask[sx, sy]
     centerline, widths = _get_centerline(cropped)
+    if centerline.size < 10:
```

Comment:
I'd actually do it as `len(centerline) < 10` for clarity. The `size` attribute is the product of the shape, so (assuming centerline has shape `(n,2)`), its `size` is twice its length. You could do `centerline.size < 20` but that's not really as clear as just saying that you want at least 10 coordinates... (which I think is about the minimum to have a sensible centerline).

Reply:
Sorry about that!